spec/features/idv/in_person_spec.rb

```diff
@@ -585,18 +580,15 @@
 
       sign_in_and_2fa_user(user)
       begin_in_person_proofing(user)
+      # prepare page
+      complete_prepare_step(user)
       complete_location_step(user)
-
-      expect(page).to have_content(
-        t('in_person_proofing.headings.prepare'), wait: 5
-      )
```

Comment:
This test appears to be failing because there is a delay in switching to the address page.

Reply:
@NavaTim from what I can tell, this test is failing because `capture_secondary_id_enabled` is being set to true in the state_id_step, which is causing all of the additional fields that are normally hidden behind that flag to be displayed. 

State ID step:  capture secondary id enabled: true
DAV in the test: false -> this is a default value
Same address in the test: true -> this is a default value

It seems that setting the config value is toggling the value in the User enrollment. I'm not sure how or why this is happening.

Reply:
The enrollment's value is determined when it's created based on the config value at that time. If you intend to test with DAV disabled, then you need to ensure it's disabled at the time that the location step is completed for that enrollment. This is by design.

That said, based on prior observations I expected that this test might be failing instead because the unload prevention dialog wasn't properly disabled before redirecting from the `prepare` page.